tests/src/integ.rs

```diff
@@ -179,16 +179,22 @@ fn it_prints_list_on_lonely_z_shell(shell: &Shell) {
 fn it_handles_existing_bash_prompt_command() {
     let tmpdir = TempDir::new("pazi_integ").unwrap();
     let root = tmpdir.path();
-    let prompt_cmd = r#"
-PROMPT_COMMAND='printf "\033k%s@%s:%s\033\\" "${USER}" "${HOSTNAME%%.*}" "${PWD/#$HOME/\~}"'
-"#;
-    let mut h = HarnessBuilder::new(&root, &Pazi, &Shell::Bash).preinit(prompt_cmd).finish();
+    let prompt_file = root.join("pazi_prompt_file");
+    let prompt_file = prompt_file.to_string_lossy();
+
+    let prompt_cmd = format!(r#"
+MY_PROMPT=1
+PROMPT_COMMAND='printf "\033k%s\033\\ \n" "${{MY_PROMPT}}" > {}'
+"#, prompt_file);
+    let mut h = HarnessBuilder::new(&root, &Pazi, &Shell::Bash).preinit(&prompt_cmd).finish();
     let slash_tmp_path = root.join("tmp");
     let slash_tmp = slash_tmp_path.to_string_lossy();
 
     h.create_dir(&slash_tmp);
     h.visit_dir(&slash_tmp);
     assert_eq!(h.jump("tmp"), slash_tmp);
+    h.run_cmd("MY_PROMPT=2");
+    assert_eq!(h.run_cmd(&format!("cat {}", prompt_file)), "2 ");
```

Comment:
I'm not sure I like this test since it passes if we strip the bash escapes...I shall look more into bash escapes in a bit, I suppose.